actix-http/Cargo.toml

```diff
@@ -97,7 +97,7 @@ open-ssl = { version="0.10", package = "openssl" }
 rust-tls = { version="0.18", package = "rustls" }
 
 [[bench]]
-name = "content-length"
+name = "write-camel-case"
```

Comment:
Why remove the old one instead of just adding the new bench?

Reply:
Since we switched to `itoa` crate it's not really a useful thing to keep track of now.